sklearn/utils/fixes.py

```diff
@@ -400,3 +400,21 @@ def rankdata(a, method='average'):
         return .5 * (count[dense] + count[dense - 1] + 1)
 else:
     from scipy.stats import rankdata
+
+
+if np_version < (1, 12, 0):
+    class MaskedArray(np.ma.MaskedArray):
+        # Before numpy 1.12, np.ma.MaskedArray object is not pickle-able
```

Comment:
I would replace pickle-able by picklable which is an acceptable adjective I think (it is definitely already mentioned a few times in the source code).